google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/application/GoogleApiClientAppEngineAdminServiceTest.java

```diff
@@ -90,8 +87,8 @@ public void testGetApplicationForProjectId_success() throws IOException, GoogleA
 
     String projectId = "my-project";
     assertEquals(result, service.getApplicationForProjectId(projectId, mock(Credential.class)));
-    verify(appengineClientMock.apps(), times(1)).get(eq(projectId));
```

Comment:
Nope, the single-param `verify()` method ensures it's only called once so this parameter is unnecessary.